corehq/apps/enterprise/enterprise.py

```diff
@@ -115,7 +123,13 @@ def total(self):
 
 
 class EnterpriseDomainReport(EnterpriseReport):
-    title = _('Project Spaces')
```

Comment:
@jingcheng16 you can just leave it as a class property. no need to do 
``
@property
def title(self):
    ...
```
etc...
```
title = ....
```
works just fine with the `ABC` class. can you make this update for readability?

Reply:
a127b2699679c7e5879737c851bc1a08c350da14